src/test/pythonnode_test.cpp

```diff
@@ -371,24 +342,20 @@ TEST_F(MediapipePythonNodeTest, PythonNodeNameDoesNotExist) {
     ASSERT_EQ(mediapipeDummy.getPythonNodeResource("pythonNode4"), nullptr);
 }
 
-TEST_F(MediapipePythonNodeTest, PythonNodeInitMembers) {
-    // Must be here - does not work when added to test::SetUp
-    // Initialize Python interpreter
-    py::scoped_interpreter guard{};  // start the interpreter and keep it alive
-    py::gil_scoped_release release;  // GIL only needed in Python custom node
+TEST_F(PythonFlowTest, PythonNodeInitMembers) {
     ConstructorEnabledModelManager manager;
     std::string testPbtxt = R"(
     input_stream: "in"
     output_stream: "out"
         node {
             name: "pythonNode2"
-            calculator: "PythonBackendCalculator"
-            input_side_packet: "PYOBJECT:pyobject"
+            calculator: "PythonExecutorCalculator"
+            input_side_packet: "PYTHON_NODE_RESOURCES:py"
             input_stream: "in"
```

Comment:
How does OVMS know that it should deserialize packet to OvmsPyTensor? Missing that indicates that it will produce ov::Tensor, doesnt it?

Reply:
You're right. I suppose multiple tests in this file do not really run the graph, so they're not failing. In that case only initialization is tested. But it would be nice to change that I agree.